.make/Makefile.dev

```diff
@@ -70,6 +75,10 @@ deploy-auth-secrets:
 deploy-wit-secrets:
 	oc process -f $(TEMPLATE_DIR)/wit.config.yaml -o yaml | oc apply -f -
 
+.PHONY: deploy-notification-secrets
```

Comment:
you could probably have the single command below in the `deploy-notification` goal, to keep the Makefile more straightforward

Reply:
I believe we already have command `deploy-notification` in the makefile, also from the name it's clear what it's doing.

Reply:
sure, but my point is: is the `deploy-notification-secret` something that we want to use independently of the `deploy-notification` goal?

Reply:
If anyone wants to just deploy secrets he can deploy it using `deploy-notification-secret` goal. Sometimes it's useful. But we are not forcing a user to run two independent goals.

Reply:
sure, I just don't see why someone would want to deploy secrets without the service, but fair-enough, we can keep it like that.